sormas-e2e-tests/src/main/java/org/sormas/e2etests/constants/api/Endpoints.java

```diff
@@ -19,8 +19,11 @@
 
 public interface Endpoints {
   String PERSONS = "persons/";
-  String CONTACTS = "contacts/";
   String CASES = "cases/";
+  String COMMUNITIES = "communities/";
+  String COUNTRIES = "countries/";
+  String FACILITIES = "facilities/";
+  String CONTACTS = "contacts/";
```

Comment:
add _PATH to all variables

Reply:
this was not added by me :). But is ok for me to do the change.